compiler/compiler.go

```diff
@@ -87,7 +88,10 @@ const (
 
 // Compile compile the target Protocol Buffers file and produces all file descriptors.
 func (c *Compiler) Compile(ctx context.Context, file *source.File, opts ...Option) ([]*descriptorpb.FileDescriptorProto, error) {
-	c.importPaths = c.importPaths[:]
```

Comment:
I didn't know that either, but unlike Python, `c.importPaths[:]` does not create a copy of a slice...